src/aquarium-optimized/Aquarium.h

```diff
@@ -518,15 +516,8 @@ class Aquarium {
   double getElapsedTime();
   void printAvgFps();
   void resetFpsTime();
-  void updateWorldMatrix(Model *model);
 
-  void updateAndDrawFishes();
-  void updateAndDrawBackground();
-
-  void updateBackground();
-  void drawBackground();
-  void updateFishes();
-  void drawFishes();
```

Comment:
Please also remove `updateWorldMatrixAndDraw()` and `updateWorldProjections()`.

Reply:
done